_scripts/app1-6.R

```diff
@@ -0,0 +1,28 @@
+{
+  
+  rm(list=ls());
+  options(show.error.locations = TRUE);
+  
+ # Prompt user to input location 
+  
+  tempLoc <- readline("Where were the measurements recorded?: ");
+  
+ # Prompt user to input recorded measurements
+  
+  temp1 <- as.numeric(readline("Enter the first measurement: "));
```

Comment:
An interesting feature of R is that it will coerce whatever you put here into a number if you use as.numeric() -- even if it is not a number.  Of course, this causes problems later.    

A more robust solution is to first check if it is a number by using is.numeric() and then cast it as a number using as.numeric().  A good use of if(), which you will be learning next lesson.